### DISCUSSION TITLE
How can I see other sites hosted along with my site using npm libraries/APIs ?

### DISCUSSION THREAD

**Discussion by Darglobe** [npm/feedback#162] [General]
Hi guys,

How can I see other sites hosted along with my site using npm libraries/APIs ?

**Comment by MylesBorins** [comment]
Hi darglobe,

unfortunately we have no way of finding out what sites are using specific libraries or APIs in production

**Comment by MylesBorins** [comment]
Hi darglobe,

unfortunately we have no way of finding out what sites are using specific libraries or APIs in production

**Comment by Darglobe** [comment]
HI Myles,

Thanks for your reply.

Basically, I was meaning what npm libraries or APIs can i use in order to
get info about all sites hosted together with my site.

Earlier it was available as one of the features of Reverse DNS lookup API,
and I was able to provide a URL of my site in the request and receive all
sites hosted on the same server.

Is there any other option to see/check list of other sites hosted together with mine on the
same server by using some available npm library or API?

Thank's in advance)
